packages/skills/botpress-skill-choice/src/views/index.jsx

```diff
@@ -81,7 +81,7 @@ export default class TemplateModule extends React.Component {
 
   onContentChanged = (element, force = false) => {
     if (element && (force || element.id !== this.state.contentId)) {
-      this.choices = _.get(element, 'data.choices') || []
+      this.choices = _.get(element, 'data.choices', _.get(element, 'choices', []))
```

Comment:
Just for my understanding why do we sometimes have `element.data.choices` and sometimes simply `element.choices`?

Reply:
`element.choices` - appeared when we weren't requesting data from server but took it from existing object on the client. Indeed that's not obvious and let me refactor that.